test/transforms.jl

```diff
@@ -859,6 +859,19 @@
     @test rt == rtₒ
   end
 
+  @testset "RowTable" begin
+    a = [3, 2, 1, 4, 5, 3]
+    b = [1, 4, 4, 5, 8, 5]
+    c = [1, 1, 6, 2, 4, 1]
+    t = Table(; a, b, c)
+    T = RowTable()
+    n, c = apply(T, t)
+    tₒ = revert(T, n, c)
+    @test typeof(t) <: Table
+    @test typeof(n) <: Vector
```

Comment:
Should also check that the new table is Tables.rowaccess

Reply:
I think this check is not necessary:
```julia
@test typeof(t) <: Table
```